src/main/java/uk/gov/pay/ledger/transaction/service/TransactionService.java

```diff
@@ -39,8 +37,9 @@
 
 public class TransactionService {
 
+    public static final String REDACTED_REFERENCE_NUMBER = "****";
```

Comment:
As this isn't actually used by the code I think it's probably best to
* remove this and update the tests to just check for the `****` string (my vote)
* use this constant in the update sql 

Probably best to not have something in there that's only used by tests